### ISSUE TITLE
Add missing foaf:focus statements to data

### ISSUE THREAD

**Issue by acka47** [hbz/lobid-resources#1029]
Similar to #1017. I stumbled over this while writing a post about NWbib data for Coding da Vinci and wanted to show that most entries have links to places in Wikidata where you can get additional information.

Task: We need to pull all the `foaf:focus` statements out of the Turtle vocab and add it to the lobid-resources data. http://lobid.org/resources/HT020170194.json is an example where the statement is missing:

```json
{
    "spatial":[
        {
            "id":"https://nwbib.de/spatial#N62",
            "type":[
                "Concept"
            ],
            "source":{
                "id":"https://nwbib.de/spatial",
                "label":"Raumsystematik der Nordrhein-Westfälischen Bibliographie"
            },
            "notation":"62",
            "label":"Herzogtum Westfalen"
        }
    ]
}
```

In the turtle file, the `foaf:focus` statements exists:

```ttl
nwbib-spatial:N62  a    skos:Concept ;
        skos:broader    nwbib-spatial:N4-7 ;
        skos:inScheme   <https://nwbib.de/spatial> ;
        skos:notation   "62" ;
        skos:prefLabel  "Herzogtum Westfalen"@de ;
        foaf:focus      wd:Q657241 .
```

**Comment by acka47** [comment]
Looks good in the test files. However, looking at this, it made me think whether adding a geo point for the larger regions (NRW, Rheinland, Westfalen, probably others) makes actually sense. (We already discussed this several times in the past. E.g. for the map on the start page (https://nwbib.de), all entries with NRW, 01 as spatial subject would be locate din Kreis Unna...  Maybe discuss a possible solution at the standup?

**Comment by dr0i** [comment]
Deployed to staging, see e.g. http://stage.lobid.org/resources/search?q=spatial.focus.id%3A%22http%3A%2F%2Fwww.wikidata.org%2Fentity%2FQ152243%22 .

**Comment by dr0i** [comment]
@acka47 as discussed yesterday at scrum, @fsteeg will filter out `focus` entries based on its type, before we can merge and deploy.

**Comment by acka47** [comment]
+1 Comparing production and stage, the former has [88018 hits with a `spatial` object and no `spatial.focus`](http://lobid.org/resources/search?q=_exists_%3Aspatial+AND+NOT+_exists_%3Aspatial.focus) while on staging there are only [818 entries like this](http://stage.lobid.org/resources/search?q=_exists_%3Aspatial+AND+NOT+_exists_%3Aspatial.focus).

**Comment by dr0i** [comment]
Re missing focus objects: opened a new issue.
Closing this one as deployed.
